aggregators/internal/hdrhistogram/hdrhistogram.go

```diff
@@ -128,7 +128,7 @@ func (h *HistogramRepresentation) Buckets() (int64, []int64, []float64) {
 			break
 		}
 		scaledCount, ok := h.CountsRep[int32(idx)]
-		if !ok {
+		if !ok || scaledCount <= 0 {
```

Comment:
[**For reviewers**] This ensures same logic as what we had before custom distribution logic was introduced.